### PR TITLE
Create copy notes.md

### PR DESCRIPTION
**Pull request by NienkeBr**
Initial documentation of some generic copy guidelines for GC digital talent.

🤖 Resolves 

## 👋 Introduction

This .md file will be tracking some decisions and notes regarding copy on the platform. I plan to continue to add on to this as more things pop-up.

### CODE REVIEW

File: documentation/copy notes.md
Comment (mnigh) [inline documentation/copy notes.md:1]: Could you rename the file to `copy.md` instead of `copy notes.md`?

File: documentation/copy notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Generally:
+-	Use infinitif for headings
+-	Use impératif for descriptions
+-	Use impératif for buttons that entail a concrete action
+
+However:
+-	Use impératif for headings that include a possessive pronoun/adjective (often found on pages that request concrete info or actions from the users)
+-	Use infinitif for buttons that are more generic. (For example that lead to more information (e.g. Learn more, Expand all, View results, Browse jobs))
+
+See for more info on the distinction between impératif and inifinitif the website of the [Office quebecois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/24208/la-grammaire/le-verbe/modes/imperatif/distinction-entre-limperatif-et-linfinitif).
```
Comment (mnigh) [inline documentation/copy notes.md:32]:
```suggestion
See for more info on the distinction between impératif and infinitif the website of the [Office quebecois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/24208/la-grammaire/le-verbe/modes/imperatif/distinction-entre-limperatif-et-linfinitif).
```

File: documentation/copy notes.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Capitalization
+[…]
+## Inclusive writing in French
+Aim to use neutral formulation whenever possible. This includes using collective nouns, gender-neutral words (nouns, adjectives, pronouns), and rephrasing when necessary.
+For more details, see the GC’s [Clés de la redaction](https://www.noslangues-ourlanguages.gc.ca/fr/cles-de-la-redaction/ecriture-inclusive-lignes-directrices-ressources) or the guidelines from the [Office québecois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/25030/banque-de-depannage-linguistique/la-redaction-et-la-communication/redaction-pour-le-web/redaction-epicene-sur-le-web).
```
Comment (mnigh) [inline documentation/copy notes.md:37]:
```suggestion
For more details, see the GC’s [Clés de la redaction](https://www.noslangues-ourlanguages.gc.ca/fr/cles-de-la-redaction/ecriture-inclusive-lignes-directrices-ressources) or the guidelines from the [Office québécois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/25030/banque-de-depannage-linguistique/la-redaction-et-la-communication/redaction-pour-le-web/redaction-epicene-sur-le-web). 
```

File: documentation/copy notes.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+-	Use impératif for descriptions
+-	Use impératif for buttons that entail a concrete action
+
+However:
+-	Use impératif for headings that include a possessive pronoun/adjective (often found on pages that request concrete info or actions from the users)
+-	Use infinitif for buttons that are more generic. (For example that lead to more information (e.g. Learn more, Expand all, View results, Browse jobs))
+
+See for more info on the distinction between impératif and inifinitif the website of the [Office quebecois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/24208/la-grammaire/le-verbe/modes/imperatif/distinction-entre-limperatif-et-linfinitif).
+## Capitalization
+[…]
```
Comment (mnigh) [inline documentation/copy notes.md:34]:
```suggestion
Coming soon.
```

Comment (mnigh) [reply documentation/copy notes.md:34]: Not sure if this is a placeholder, but better to say what it is than [...] which is markdown.

**Review by mnigh — CHANGES_REQUESTED** [review]
A few copy suggestions, but otherwise, looks great. 👍

File: documentation/copy notes.md (hunk 3 above)
Comment (NienkeBr) [reply documentation/copy notes.md]: good point, removed it altogether and will re-add when I have something concrete.

**Review by mnigh — APPROVED** [review]
Congratulations on your first PR, nice work. 🎉

**Pull request merged**